staging/src/k8s.io/kubectl/pkg/kuberc/marshal.go

```diff
@@ -40,6 +40,12 @@ func decodePreference(kubercFile string) (*config.Preference, error) {
 		return nil, err
 	}
 
+	// olderVersions map holds valid preference contents in older, known versions.
+	// The current version priorities are:
+	// 1. v1beta1
+	// 2. v1alpha1
+	// When decoding versions, we always try to pick the latest known version.
```

Comment:
hmm... it's a little confusing to describe the "first usable one we find in the file" with "latest API version" ... is there a reason not to just keep the "first usable one we find in the file" approach and let them put v1beta1 first if that's what they want newer kubectls to use?

Reply:
Hmmm... I think you're right. That will be definitely much easier to explain.